xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-ui/src/main/resources/CKEditor/ConfigSheet.xml

```diff
@@ -218,15 +218,19 @@ require(['jquery', 'xwiki-ckeditor', 'bootstrap-select'], function($, ckeditorPr
   var commaSeparator = /\s*,\s*/;
   var enhanceConfig = function(editor) {
     var availablePlugins = editor.config.plugins.split(commaSeparator);
-    enhanceList($('#CKEditor\\.ConfigClass_0_removePlugins'), availablePlugins);
+    enhanceList($('#CKEditor\\.ConfigClass_0_removePlugins'), availablePlugins, 'availablepluginsselect');
```

Comment:
I'm not aware of any naming convention for ids, but I think it would be nice to use at least some way to differentiate words like camelCase.

Reply:
You are right. I don't know of one either.
When checking the string `id="` in the codebase, it prompted a lot of things, but the most regularly used convention for IDs was camelCase.

I created a forum proposal to add a HTML codestyle appropriate for this situation: https://forum.xwiki.org/t/html-codestyle-proposal-conventions-for-ids-and-classes/13619

Reply:
So apparently the proposal went for the kebab-case for ids so I guess you need to fix that now :)

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2581/commits/8599be3eb3639c067095378f06bbdd7431bf0844 :+1: